pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -32,6 +34,74 @@ var info = capabilities.MustNewCapabilityInfo(
 	"OCR3 consensus exposed as a capability.",
 )
 
+type Output = any
+
+type CapabilityConfig struct {
+	AggregationMethod string `json:"aggregationMethod" jsonschema:"required"`
+	AggregationConfig any    `json:"aggregationConfig" jsonschema:"required"`
+	Encoder           string `json:"encoder" jsonschema:"required"`
+	EncoderConfig     any    `json:"encoderConfig" jsonschema:"required"`
+	ReportID          string `json:"reportId" jsonschema:"required"`
+}
+
+type CapabilityInputs struct {
+	// This Input is coupled to the Datastreams Trigger. This was an
+	// intentional short-term trade-off that we'll fix in the future.
+	Observations workflows.CapabilityDefinition[[]datastreams.FeedReport]
+}
+
+type NewOCR3ConsensusParams struct {
+	Inputs CapabilityInputs
+	Config CapabilityConfig
+}
+
+// type NewMercuryTriggerParams struct {
+// 	Config Config
+// }
+
+// triggers.NewMercuryTriggerParams{
+// 	Config: triggers.Config{
+// 		FeedIDs: []string{
+// 			"0x0003fbba4fce42f65d6032b18aee53efdf526cc734ad296cb57565979d883bdd",
+// 			"0x0003c317fec7fad514c67aacc6366bf2f007ce37100e3cddcacd0ccaa1f3746d",
+// 			"0x0003da6ab44ea9296674d80fe2b041738189103d6b4ea9a4d34e2f891fa93d12",
+// 		},
+// 		MaxFrequencyMs: 100,
+// 	},
+// },
+
+func NewOCR3Consensus(params NewOCR3ConsensusParams) workflows.Consensus[Output] {
+	// // TODO: Call .ValidateConfig to check for more complex JSON Schema validation
+	// outputs := []datastreams.FeedReport{}
+
+	// for _, feedID := range params.Config.FeedIDs {
+	// 	outputs = append(outputs, datastreams.FeedReport{
+	// 		FeedID: feedID,
+	// 	})
+	// }
```

Comment:
This is a hack because we don't have parallel composition, right?  As far as I can tell, there's no way to know to generate it.